kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js

```diff
@@ -306,12 +314,15 @@ define([
                     render: function (data) {
                         const fileId = new UUID(4).format();
                         //render checkboxes disabled until the user selects a type
-                        return ('<input class="kb-staging-table-body__checkbox-input"' + 
-                        'type="checkbox" disabled=true' + 
-                        'aria-checked="false" tabindex="0"' +
-                        'aria-label="Select to import file checkbox: disabled until at least one data type is selected"' +
-                        'data-file-name="' + data + '"' + 
-                        'id="' + fileId + '">');
+                        return ('<div class="kb-staging-table-body__checkbox-disabled" ' + 
```

Comment:
have to wrap the checkbox inputs in a div so we can have the tooltip (otherwise it won't display on disabled elements)